ci/vm.sh

```diff
@@ -353,6 +354,11 @@ do_test() {
 	rm -f uploads/*
 
 	color=34
+	colorize $color "== Running Boot & Phone-Home Test =="
+	test_boot_and_phone_home |& stdbuf -i 0 sed "s/^/$(colorize $color 'test_boot_and_phone_home│')/" && echo "this test is expected to fail" >&2 && exit 1
```

Comment:
break this statement out into multiple lines?

Reply:
Glad you called me out on this because I had some other stuff laying around in a stash that I actually wanted to have in the PR :D.